### ISSUE TITLE
Plugin incompatible with PaperMC version 1.21

### ISSUE THREAD

**Issue by Umbra100** [Efnilite/WorldServer#10]
**Describe the bug**
When starting a Paper server (1.21) with the WorldServer plugin (1.9.3), the plugin encounters an exception and disables itself.

**To Reproduce**
1. Create new server instance of Paper with version 1.21
2. Download WS off of the Spigot plugins website (v1.9.3) and put the jar executable in the plugins folder
3. Run the server and see the error created.

**Screenshots**
![Screenshot 2024-07-22 155340](https://github.com/user-attachments/assets/66cf6afd-dfc5-4e9c-a708-c7b33c77eade)

**Errors**
```
[15:20:28] [Server thread/ERROR]: Error occurred while enabling WorldServer v1.9.3 (Is it up to date?)
java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3
	at WorldServer-1.9.3.jar/dev.efnilite.ws.vilib.util.Version.getInternalVersion(Version.java:64) ~[WorldServer-1.9.3.jar:?]
	at WorldServer-1.9.3.jar/dev.efnilite.ws.vilib.util.Version.getVersion(Version.java:51) ~[WorldServer-1.9.3.jar:?]
	at WorldServer-1.9.3.jar/dev.efnilite.ws.vilib.ViPlugin.onEnable(ViPlugin.java:45) ~[WorldServer-1.9.3.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:288) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[paper-1.21.jar:1.21-108-73a863b]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.21.jar:1.21-108-73a863b]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:640) ~[paper-1.21.jar:1.21-108-73a863b]
	at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:589) ~[paper-1.21.jar:1.21-108-73a863b]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:753) ~[paper-1.21.jar:1.21-108-73a863b]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:515) ~[paper-1.21.jar:1.21-108-73a863b]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:329) ~[paper-1.21.jar:1.21-108-73a863b]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1214) ~[paper-1.21.jar:1.21-108-73a863b]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:330) ~[paper-1.21.jar:1.21-108-73a863b]
	at java.base/java.lang.Thread.run(Thread.java:1570) ~[?:?]
```

**Versions**
- Plugin version: 1.9.3
- Paper version: 1.21

**Additional context**
The error is only created using Paper; when running this plugin on Spigot, no errors are thrown.

**Comment by lelongm** [comment]
Hello i've got the same error

**Comment by Yurielvillanueva** [comment]
same error too.

**Comment by Efnilite** [comment]
Fixed in 1.9.4. Sorry for the delay!
